docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -42,7 +41,44 @@ docker pull docker.elastic.co/beats/elastic-agent-complete:{version}
 ----
 
 [discrete]
-== Step 2: Get aware of the  {agent} container command
+== Step 2: Optional: Verify the image
+
+Although it's optional, we highly recommend verifying the signatures included with your downloaded Docker images to ensure that the images are valid.
+
+Elastic images are signed with https://docs.sigstore.dev/cosign/overview/[Cosign] which is part of the https://www.sigstore.dev/[Sigstore] project. Cosign supports container signing, verification, and storage in an OCI registry. Install the appropriate https://docs.sigstore.dev/cosign/installation/[Cosign application]
+for your operating system.
+
+Run the following commands to verify the *elastic-agent* container image signature for {agent} v{version}:
+
+["source","sh",subs="attributes"]
+--------------------------------------------
+wget https://artifacts.elastic.co/cosign.pub <1>
+cosign verify --key cosign.pub docker.elastic.co/beats/elastic-agent:{version} <2>
+--------------------------------------------
+<1> Download the Elastic public key to verify container signature
+<2> Verify the container against the Elastic public key
+
+If you're using the *elastic-agent-complete* image, run the commands as follows:
+
+["source","sh",subs="attributes"]
+--------------------------------------------
+wget https://artifacts.elastic.co/cosign.pub <1>
+cosign verify --key cosign.pub docker.elastic.co/beats/elastic-agent-complete:{version} <2>
+--------------------------------------------
+
+The command prints the check results and the signature payload in JSON format, for example:
+
+[source,sh]
```

Comment:
I think this is needed to swap our `{version}`.
```suggestion
["source","sh",subs="attributes"]
```

Reply:
Nice catch @jrodewig!